godot-core/src/docs.rs

```diff
@@ -79,20 +79,24 @@ pub fn gather_xml_docs() -> impl Iterator<Item = String> {
 
             let InherentImplDocs {
                 methods,
-                signals,
-                constants,
+                signals_block,
+                constants_block,
             } = pieces.inherent;
 
             let virtual_methods = pieces.virtual_methods;
             let brief = description.split_once("[br]").map(|(x, _)| x).unwrap_or_default();
+            let methods_block: String = match methods.is_some() || virtual_methods.is_some() {
+                true => format!("<methods>{m}{vm}</methods>", m=methods.unwrap_or_default(), vm=virtual_methods.unwrap_or_default()),
+                _ => "".to_string()
+            };
```

Comment:
Use `if` instead of `match`

Also, `String::new()` instead of `"".to_string()`

Reply:
done